packages/suite/src/hooks/wallet/sign-verify/useSignVerifyForm.ts

```diff
@@ -95,6 +99,11 @@ export const useSignVerifyForm = (page: 'sign' | 'verify', account?: Account) =>
         },
     });
 
+    const { field: isElectrumField } = useController({
```

Comment:
I noticed that when you successfully sign a message and then switch `isElectrum` value, you're not able to re-sign with new options. I believe it could be fixed by adding `isElectrum` to one of the `useEffect`s below.

Reply:
done